src/olympia/addons/models.py

```diff
@@ -1305,6 +1305,17 @@ def all_categories(self):
         return filter(
             None, [cat.to_static_category() for cat in self.categories.all()])
 
+    @cached_property
+    def current_previews(self):
```

Comment:
Where do we still use `Addon.all_previews`?  Could we just switch to `current_previews` everywhere and remove the `all_previews` method?

Reply:
It's doable but I wanted to avoid touching the `transformer` for the moment... We still need something that it can write to to "prefetch" previews.